src/userProfile/UserProfileContext.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from "react";
+
+export default createContext(null);
```

Comment:
This is causing an error in the browser as you are trying to access user.username in another components